app/src/main/java/org/mozilla/fenix/compose/SectionHeader.kt

```diff
@@ -28,41 +27,14 @@ fun SectionHeader(
     modifier: Modifier = Modifier
 ) {
     Text(
-        modifier = modifier,
         text = text,
-        style = TextStyle(
-            fontFamily = FontFamily(Font(R.font.metropolis_semibold)),
-            fontSize = 20.sp,
-            lineHeight = 20.sp
-        ),
-        maxLines = 1,
-        overflow = TextOverflow.Ellipsis,
-        color = FirefoxTheme.colors.textPrimary
-    )
-}
-
-/**
- * Default layout for the header of a screen section.
- *
- * @param text [String] to be styled as header and displayed.
- * @param modifier [Modifier] to be applied to the [Text].
- */
-@Composable
-fun HomeSectionHeader(
```

Comment:
Mostly, looking to de-dupe since the only change was the `fontSize`. Otherwise, `SectionHeader` is technically unused.